libcxx/test/std/containers/sequences/vector.bool/shrink_to_fit.pass.cpp

```diff
@@ -39,9 +39,48 @@ TEST_CONSTEXPR_CXX20 bool tests()
     return true;
 }
 
+#if TEST_STD_VER >= 23
+std::size_t min_bytes = 1000;
```

Comment:
[This comment](https://github.com/llvm/llvm-project/pull/97895#discussion_r1678057695) applies here too!